src/main/java/core/basesyntax/impl/ValidatorServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ValidatorService;
+import java.util.List;
+
+public class ValidatorServiceImpl implements ValidatorService {
+    private static final String TITLE_TEXT = "type,fruit,quantity";
+    private static final String FORMAT_TEXT = "[bpsr]{1},[a-z]{3,},\\d{1,}";
+
+    public Boolean isValid(List<String> text) {
```

Comment:
```suggestion
    public boolean isValid(List<String> text) {
```
I think this will be enough